### PR TITLE
Accounts page with different components version

### PR DESCRIPTION
**Pull request by acdbaykal**
@davidyuk,

I made minimal change to your code to integrate https://github.com/aeternity/aepp-components/tree/update-identities into your PR (#90 )

### CODE REVIEW

File: src/components/QuickId.vue [hunk 1]
```diff
@@ -37,15 +42,31 @@
...
+    .vertical-ruler {
+      margin: 0 15px;
+      height: 25px;
+      border-right: 1px solid #ffffff;
+      align-self: center;
+    }
+
+    .quick-id__chevron {
+      fill: #ffffff;
+      transform: rotate(-90deg);
```
Comment (davidyuk) [inline src/components/QuickId.vue]: AeIcon has `rotate` property, probably better to use it.

File: src/components/QuickId.vue [hunk 2]
```diff
@@ -37,15 +42,31 @@
...
+    }
+
+    .vertical-ruler {
+      margin: 0 15px;
+      height: 25px;
+      border-right: 1px solid #ffffff;
+      align-self: center;
+    }
+
+    .quick-id__chevron {
```
Comment (davidyuk) [inline src/components/QuickId.vue]: I think `quick-id__` should be omitted as far as this class already inside node with class `quick-id`.

File: src/components/QuickId.vue [hunk 3]
```diff
@@ -1,22 +1,26 @@
 <template>
-  <ae-identity class="quick-id" :identity="identity" active @click="showIdManager" collapsed>
+  <ae-account-background class="quick-id" type="dramatic"  @click="showIdManager">
```
Comment (davidyuk) [inline src/components/QuickId.vue]: Two spaces after `type="dramatic"`.

File: src/components/QuickId.vue [hunk 4]
```diff
@@ -25,6 +29,7 @@
     },
     methods: {
       showIdManager () {
+        console.log('showIdManager')
```
Comment (davidyuk) [inline src/components/QuickId.vue]: Please, remove this `console.log`.

**Comment by davidyuk** [comment]
It looks broken when I choose the second account in inactive accounts list.

screenshot from 2018-04-10 11-11-44

File: src/components/QuickId.vue [hunk 5]
```diff
@@ -1,22 +1,26 @@
...
+  <ae-account-background class="quick-id" type="dramatic"  @click="showIdManager">
     <router-link
       :to="{ name: 'apps' }"
-      class="back"
-      slot="header-left"
-      v-if="showBackButton"
       @click.native.stop
-    />
-  </ae-identity>
+      class="back-link"
```
Comment (davidyuk) [inline src/components/QuickId.vue]: `back-link` is not mentioned anywhere.

File: src/components/QuickId.vue [hunk 6]
```diff
@@ -37,15 +42,31 @@
...
+    }
+
+    .quick-id__chevron {
+      fill: #ffffff;
+      transform: rotate(-90deg);
+      cursor: pointer;
+    }
+
+    .quick-id__active-identity{
+      pointer-events: none;
```
Comment (davidyuk) [inline src/components/QuickId.vue]: I think `cursor: default` is more obvious. Or probably we can apply `cursor: pointer` for the whole `.quick-id`.

**Comment by acdbaykal** [comment]
@davidyuk,

I could not reproduce the bug in Firefox or Chrome. Which browser were you using?

[](https://postimg.cc/image/yih1wg26z/)
[](https://postimg.cc/image/5rk86d1i3/)

**Comment by davidyuk** [comment]
> I could not reproduce the bug in Firefox or Chrome. Which browser were you using?

This bug are maked in #90, it is fixed in f242f11af671dea2011bb5c62549334601d68ce8

**Comment by davidyuk** [comment]
It should not be closed

**Comment by acdbaykal** [comment]
Replaced by https://github.com/aeternity/aepp-identity/pull/103

**Pull request closed without merging**